examples/redis-operator.md

```diff
@@ -0,0 +1,109 @@
+# Redis-pmem operator
+This a complete example to deploy a Redis cluster through the [redis-operator](https://github.com/spotahome/redis-operator) using QEMU-emulated persistent memory devices.
+
+## PMEM-CSI installation
+The steps to install PMEM-CSI within a K8s cluster with one master and three workers nodes using a fix version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced, are listed below:
+1. Clone this project into `$GOPATH/src/github.com/intel`
+2. Build PMEM-CSI:
+    ```sh
+     $ cd $GOPATH/src/github.com/intel/pmem-CSI
+     $ make push-images
+    ```
+    **Note**: by default the build images are going to be pushed into a local [Docker registry](https://docs.docker.com/registry/deploying/).
+
+3. Start the K8s cluster
+    ```sh
+    $ TEST_CLEAR_LINUX_VERSION=29820 make start
+    ```
+    **WARNING**: you may run into a SSL problem when executing this command, which can be avoided by disabling the signed files test checks, so the final command to be executed will be: `TEST_CLEAR_LINUX_VERSION=29820 TEST_CHECK_SIGNED_FILES=false make start`, be aware that **this is not a secure step**.
+
+4. Setup `KUBECONFIG` env variable to use `kubectl` binary
+    ```sh
+    $ export KUBECONFIG=$(pwd)/_work/clear-govm/kube.config
+    ```
+5. Verify that all pods reach `Running` status
+    ```sh
+    $ kubectl get po
+    NAME                          READY   STATUS    RESTARTS   AGE
+    pmem-csi-controller-0         3/3     Running   0          125ms
+    pmem-csi-node-c82p5           2/2     Running   0          124m
+    pmem-csi-node-dwws2           2/2     Running   0          125m
+    pmem-csi-node-kj7gs           2/2     Running   0          125m
+    ```
+
+## Redis operator installation and Redis cluster deployment
+The steps to install the Redis operator by Spotahome are listed below:
+ 1. Clone the [redis-operator](https://github.com/spotahome/redis-operator) project into `$GOPATH/src/github.com/spotahome`
```

Comment:
Why do you need to clone. Can you not apply directly from github

```kubectl apply -f https://raw.githubusercontent.com/spotahome/redis-operator/master/example/operator/all-redis-operator-resources.yaml```

Reply:
I think it was easier to clone the repo to access `example/operator/all-redis-operator-resources.yaml` and `example/redisfailover/pmem.yaml`, and _potentially_ a user could modify `all-redis-operator-resources.yaml` and deploy more redisfailover exampes.

Reply:
The new patch creates the operator and deploys the Redis cluster using raw files as @mcastelino suggested.